libpod/info.go

```diff
@@ -0,0 +1,120 @@
+package libpod
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/docker/docker/pkg/system"
+)
+
+type InfoData struct {
```

Comment:
I would really prefer it if we separated the error out of this, and just made it type and map. we can return the error separately.

Reply:
fixed